### ISSUE TITLE
Don't reset authid on login call, zabbix.js line 103

### ISSUE THREAD

**Issue by jykae** [pawmint/zabbix.js#6]
I removed this line to login just once in my application. 
Otherwise I would need to login each time before I make a call to Zabbix API.

**Comment by tibotiber** [comment]
@jykae: thanks for the report. 
@joaobentes: [this is from you](https://github.com/pawmint/zabbix.js/blob/master/lib/zabbix.js#L103), can you clarify? Is that part of the code you wanted to rollback and refactor?

**Comment by joaobentes** [comment]
@tibotiber No, it comes with the zabbix.js. I had proposed to code a kind of safe call, when the login is made before the call and a logout is made after the call. However, such approach did not solve my issue and seemed too heavy. Basically, the solution was to login only once (when the hook of our system starts).
